### DISCUSSION TITLE
Current state of the refactoring

### DISCUSSION THREAD

**Discussion by GhostInTheProd** [OPEN-AIR-SUN/mars#83] [Q&A]
Hello, 

I really enjoyed reading your paper and would like to use your approach for a research project. I was wondering, is there any update on the current state of the refactoring? When will it be done and how big will the impact of the refactoring be? 

Thanks in advance!

**Comment by wuzirui** [comment]
Thank you for your interest. The code in #15 should be working though it has not been tested thoroughly yet.

Issues/discussions & PRs are welcomed when you're using that version.
